### DISCUSSION TITLE
GUI Tiny11 Creator

### DISCUSSION THREAD

**Discussion by SUP2Ak** [ntdevlabs/tiny11builder#297] [Show and tell]
![IMG_3233](https://github.com/user-attachments/assets/b1e657d1-884c-4373-ad4f-5aa3b248379c)

Hello everyone :),

I created a desktop interface for running the Tiny11 script, allowing users to select what they want to include or exclude when creating the ISO.

**Comment by Paloik** [comment]
> ![IMG_3233](https://private-user-images.githubusercontent.com/31973315/393870284-b1e657d1-884c-4373-ad4f-5aa3b248379c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYzMzk3NzAsIm5iZiI6MTczNjMzOTQ3MCwicGF0aCI6Ii8zMTk3MzMxNS8zOTM4NzAyODQtYjFlNjU3ZDEtODg0Yy00MzczLWFkNGYtNWFhM2IyNDgzNzljLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA4VDEyMzExMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA2ZGExMGM1NWZiMmU5Y2M1NTE2NmEyODYwMGJlN2FiYzg3ZjEzNzg3ZmU5OTk3NDBkNTMyNjg0ZDAzMmMyOGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wbSgLKxKzi8ERudApBUo_2iRwD4ZMof7w0fD6-evvSk)
> 
> Hello everyone :),
> 
> I created a desktop interface for running the Tiny11 script, allowing users to select what they want to include or exclude when creating the ISO.

hej, i would like to try it. Can you please tell me where to find a working downloadbale version of your GUI?

**Reply by pitekkate** [reply]
> > ![IMG_3233](https://private-user-images.githubusercontent.com/31973315/393870284-b1e657d1-884c-4373-ad4f-5aa3b248379c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYzMzk3NzAsIm5iZiI6MTczNjMzOTQ3MCwicGF0aCI6Ii8zMTk3MzMxNS8zOTM4NzAyODQtYjFlNjU3ZDEtODg0Yy00MzczLWFkNGYtNWFhM2IyNDgzNzljLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA4VDEyMzExMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA2ZGExMGM1NWZiMmU5Y2M1NTE2NmEyODYwMGJlN2FiYzg3ZjEzNzg3ZmU5OTk3NDBkNTMyNjg0ZDAzMmMyOGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wbSgLKxKzi8ERudApBUo_2iRwD4ZMof7w0fD6-evvSk)
> > Hello everyone :),
> > I created a desktop interface for running the Tiny11 script, allowing users to select what they want to include or exclude when creating the ISO.
> 
> hej, i would like to try it. Can you please tell me where to find a working downloadbale version of your GUI?

See PR https://github.com/Miiraak/tiny11builderGUI/tree/PR-GUI

**Reply by TheBeardofKnowledge** [reply]
Same here, maybe even make a video on it in my socials if it's ready for production preview...

**Comment by SUP2Ak** [comment]
> > ![IMG_3233](https://private-user-images.githubusercontent.com/31973315/393870284-b1e657d1-884c-4373-ad4f-5aa3b248379c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYzMzk3NzAsIm5iZiI6MTczNjMzOTQ3MCwicGF0aCI6Ii8zMTk3MzMxNS8zOTM4NzAyODQtYjFlNjU3ZDEtODg0Yy00MzczLWFkNGYtNWFhM2IyNDgzNzljLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA4VDEyMzExMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA2ZGExMGM1NWZiMmU5Y2M1NTE2NmEyODYwMGJlN2FiYzg3ZjEzNzg3ZmU5OTk3NDBkNTMyNjg0ZDAzMmMyOGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wbSgLKxKzi8ERudApBUo_2iRwD4ZMof7w0fD6-evvSk)
> 
> > 
> 
> > Hello everyone :),
> 
> > 
> 
> > I created a desktop interface for running the Tiny11 script, allowing users to select what they want to include or exclude when creating the ISO.
> 
> 
> 
> hej, i would like to try it. Can you please tell me where to find a working downloadbale version of your GUI?

Hello buddy, currently this project is on my old computer, I will have to recover this project within a week on my old hard drive, this project is made in Rust, I can make sure you get a json or other file (key value) to configure the options, as well as customize executable to your powershell script if you want, in any case I push and put open source GUI if you want to do that directly in the source code.
